src/api/app/controllers/webui/request_controller.rb

```diff
@@ -337,4 +342,20 @@ def request_action_attributes(type)
     opt['type'] = type.to_s
     opt
   end
+
+  def render_request_update
+    render partial: 'update', locals: {
```

Comment:
Is there any benefit to have this method only for render partial?

Reply:
If I will put it to `show` method, then linter will complain, that there are too many lines in the method. Could you please tell me if you have any other idea how can I solve that issue? Thank you.

Reply:
I see now, `show` action is a little big :sweat_smile:. Maybe doing it in other way, moving all those instance variables to a method like for example: `load_dependencies`. But right now, I don't have a better solution for the issue, sorry :crying_cat_face: